package.json

```diff
@@ -18,13 +18,10 @@
 	"main": "index.js",
 	"jest": {
 		"collectCoverage": false,
-		"collectCoverageFrom": [
-			"<rootDir>/*.js"
-		],
 		"roots": [
 			"<rootDir>/test"
 		],
-		"testRegex": "\\.js"
```

Comment:
I believe that anything matching the testRegex is getting excluded from coverage.

It defaults to anything in __tests__ or any file with `.test` or `.spec`. 
```
(/__tests__/.*|(\\.|/)(test|spec))\\.jsx?$
```